server/client.go

```diff
@@ -2677,6 +2716,69 @@ func (c *client) getRTTValue() time.Duration {
 	return rtt
 }
 
+// Creates a cache suitable for ROUTER and GATEWAY connections to
+// retrieve subject interest on a given account.
+func createPerAccountCache() map[string]*perAccountCache {
+	return make(map[string]*perAccountCache, maxPerAccountCacheSize)
```

Comment:
Again if one liner do we need it to be its own func?

Reply:
I wasn't sure of the name of the stuct.. so that would have been one place to change it. Will change.. although compiler may inline, but anyway.

Reply:
done